### PR TITLE
Added AniTube Presence

### PR DESCRIPTION
**Pull request by Yuujet**
image

### CODE REVIEW

File: AniTube/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "id": "316989384077475841"
+  },
+  "service": "AniTube",
+  "description": {
+    "en": "Watching Animes Online for free has never been easier, on Anitube you can see Naruto Shippuden, Bleach Online, all of their Anime Subtitled.",
+    "pt": "Assistir Animes Online gratuitamente nunca foi tão fácil, no Anitube você pode ver Naruto Shippuden, Bleach Online, todos os seus Anime legendados.",
+    "pt_BR": "Assistir Animes Online gratuitamente nunca foi tão fácil, no Anitube você pode ver Naruto Shippuden, Bleach Online, todos os seus Anime legendados."
+  },
+  "url": "www.anitube.site",
+  "version": "1.0",
```
Comment (ririxidev) [inline AniTube/dist/metadata.json:14]: 1.0.0

File: AniTube/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "logo": "https://i.imgur.com/RkIE0qc.png",
```
Comment (ririxidev) [inline AniTube/dist/metadata.json:15]: min. 512x512

File: AniTube/dist/presence.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+                if (video.paused) {
+                    delete presenceData.startTimestamp;
+                    delete presenceData.endTimestamp;
+                }
+            }
+        }
+        presence.setActivity(presenceData);
+        return [2 /*return*/];
+    });
+}); });
```
Comment (ririxidev) [inline AniTube/dist/presence.js:140]: delete this file

File: AniTube/presence.ts [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+ */
+
+function getTimestamps(
+  videoTime: number,
+  videoDuration: number
+): Array<number> {
+  var startTime = Date.now();
+  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  return [Math.floor(startTime / 1000), endTime];
+}
```
Comment (ririxidev) [inline AniTube/presence.ts:22]: you should delete this function and use presence.getTimestamps

File: AniTube/presence.ts [hunk 5]
```diff
@@ -0,0 +1,126 @@
+var presence = new Presence({
```
Comment (ririxidev) [inline AniTube/presence.ts:1]: no vars, change to const

File: AniTube/presence.ts [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+  videoDuration: number
+): Array<number> {
+  var startTime = Date.now();
+  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  return [Math.floor(startTime / 1000), endTime];
+}
+
+var lastPlaybackState = null;
+var playback;
+var browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (ririxidev) [inline AniTube/presence.ts:26]: no vars, change to const/let and merge them

File: AniTube/presence.ts [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+if (lastPlaybackState != playback) {
+  lastPlaybackState = playback;
+  browsingStamp = Math.floor(Date.now() / 1000);
+}
+presence.on("UpdateData", async () => {
+
+    const presenceData: presenceData = {
+      largeImageKey: "at"
+    };
+  const tempo = Math.floor(Date.now() / 1000),
```
Comment (ririxidev) [inline AniTube/presence.ts:37]: merge consts

File: AniTube/presence.ts [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+  const video: HTMLVideoElement = document.querySelector(".jw-video");
+
+  if (video !== null && !isNaN(video.duration)) {
+    var videoTitle;
+    var seasonepisode;
+
+    videoTitle = AniN;
+    seasonepisode = AniEp.replace("– ", "");
+
+    var timestamps = getTimestamps(
```
Comment (ririxidev) [inline AniTube/presence.ts:101]: no vars

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
ah, and this folder should be in ``websites`` folder not in root folder :P

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix sv and codacy issues

**Comment by ririxidev** [comment]
one pr or nothing. decide

**Pull request closed without merging**